roles/openshift_logging/defaults/main.yml

```diff
@@ -13,10 +13,26 @@ openshift_logging_curator_run_minute: 0
 openshift_logging_curator_run_timezone: UTC
 openshift_logging_curator_script_log_level: INFO
 openshift_logging_curator_log_level: ERROR
+openshift_logging_curator_cpu_limit: 100m
+openshift_logging_curator_memory_limit: null
```

Comment:
why are we creating defaults for these at all if they're null? Isn't that the same as leaving them off?